packages/mui-material/src/Popper/Popper.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import PopperUnstyled, { PopperUnstyledProps } from '@mui/core/PopperUnstyled';
+import { chainPropTypes, HTMLElementType, refType } from '@mui/utils';
+import { Direction, useThemeWithoutDefault as useTheme } from '@mui/system';
+
+export type PopperProps = PopperUnstyledProps;
```

Comment:
I think we should omit the direction prop from the unstyled component

Reply:
Does it do any harm, though? It would be a way to override the theme's value if needed.

Reply:
If we want to support this, we should add it on all components that depend on it, having it on only one components can create confusion as to why it is supported only here.

Reply:
I omitted the direction from the props but it still shows up on the API docs (because the unstyled component uses it)... This automatic prop 'inheritance' is nice sometimes, but not always :)

Reply:
:( Let's keep it then for now. It's worse to not support it in the typings, but show it on the docs API page. And we need to fix this in the future

Reply:
There may be cases in the future where we want to restrict the props for some components in the design system packages compared to the unstyled.

Reply:
Right, we'll have to rework the API docs generation scripts.

Reply:
Let's add TODO please so that we don't forget about it.

Reply:
I'll create an issue for even better visibility.